### ISSUE TITLE
[BUG] Last two month are not successfully transferred to a new line when displaying a full year calendar on a wide screen

### ISSUE THREAD

**Issue by kudrin** [uvarov-frontend/vanilla-calendar-pro#339]
**⚠️ Important: Bug reports are accepted only for version 3.0.0 or above**
Before submitting a bug report, ensure that you are using at least version **3.0.0** of Vanilla Calendar Pro. Issues reported for earlier versions will not be addressed.

---

**Describe the bug**
December and November are not successfully transferred to a new line when displaying a full year calendar on a wide screen

**Steps to reproduce**
```js
import { Calendar, type Options } from "vanilla-calendar-pro";

import "vanilla-calendar-pro/styles/index.css";

const options: Options = {
  type: "multiple", // Display multiple months at once
  displayMonthsCount: 12, // Display 12 months
  monthsToSwitch: 1, // Disabling month jump
};

const calendar = new Calendar("#calendar", options);
calendar.init();
```

**CodeSandbox example (REQUIRED)**
Still failed

**Screenshots (optional)**
![image](https://github.com/user-attachments/assets/9f793e38-bb79-4ef9-8232-bb4d4c0df585)

**Comment by uvarov-frontend** [comment]
If you mean that the months stretch to the full width, then that’s how it’s intended. You need to customize your styles to get rid of this effect.

**Comment by kudrin** [comment]
Yeah, I mean that. But it was fine on 2.x version on default style.

**Comment by uvarov-frontend** [comment]
Вы можете изменить стили как вариант вот так: 
```css
[data-vc='grid'] {
  display: grid;
  grid-template-columns: repeat(5, 1fr);
}
```
